go.mod

```diff
@@ -60,7 +60,7 @@ require (
 	github.com/jbenet/go-temp-err-catcher v0.1.0
 	github.com/jbenet/goprocess v0.1.4
 	github.com/libp2p/go-doh-resolver v0.3.1
-	github.com/libp2p/go-libp2p v0.14.2
+	github.com/libp2p/go-libp2p v0.14.3
```

Comment:
Can we get v0.14.4 in (see https://github.com/ipfs/go-ipfs/pull/8267#pullrequestreview-708861874), mostly to avoid confusion later on which version we were actually running?

Reply:
It seems like the only thing added, aside from the TCP bump, was https://github.com/libp2p/go-libp2p/pull/1115.

@Stebalien are you happy letting that into the patch release, or do you think we're better off just waiting on it?

Reply:
I'd really rather not include that. I've mostly convinced myself it won't cause problems, but still.

Reply:
No worries, let's just leave it alone for now and we'll catch it in the next release